app.json

```diff
@@ -0,0 +1,44 @@
+{
+  "expo": {
+    "name": "Movement-Vites",
```

Comment:
Lets make all the names Movement-Vitamins as I am pretty sure Cory is pursuing a copyright for that name

Reply:
Sorry, I must have missed that one